src/graphql/schemas/ParseClass.js

```diff
@@ -462,6 +476,56 @@ export class ParseClass {
     };
   }
 
+  graphQLSubscriptionConfig() {
+    const className = this.className;
+    return {
+      name: this.displayName + 'Subscription',
+      description: `Parse Class ${className} Subscription`,
+      fields: () => {
+        const objectFields = this.buildFields(graphQLQueryField);
+        delete objectFields.objectId;
+        delete objectFields.id;
+        return {
+          op : 
+          object : objectFields
```

Comment:
the subscription takes I believe a list of operations from your enum (options) and a filter (the query type for this class).

The subcrition emits an object that is:

```
 {
   event: EventType,
   node: objectType 
}
```